src/test/java/org/apache/drill_web_test_framework/web_ui/steps/NavigationSteps.java

```diff
@@ -16,23 +16,33 @@
  */
 package org.apache.drill_web_test_framework.web_ui.steps;
 
-import org.apache.drill_web_test_framework.web_ui.pages.BasePage;
+import org.apache.drill_web_test_framework.web_ui.pages.ChooseAuthMethodPage;
 import org.apache.drill_web_test_framework.web_ui.pages.NavigationPage;
 import org.apache.drill_web_test_framework.web_ui.pages.StoragePage;
 import org.apache.drill_web_test_framework.web_ui.pages.ProfilesPage;
 
+import static org.apache.drill_web_test_framework.web_ui.pages.BasePage.getPage;
+
 public final class NavigationSteps extends BaseSteps {
 
   public QuerySteps navigateQuery() {
-    BasePage.getPage(NavigationPage.class).navigateQuery();
+    getPage(NavigationPage.class).navigateQuery();
     return BaseSteps.getSteps(QuerySteps.class);
   }
 
   public StoragePage navigateStorage() {
-    return BasePage.getPage(NavigationPage.class).navigateStorage();
+    return getPage(NavigationPage.class).navigateStorage();
   }
 
   public ProfilesPage navigateProfiles() {
-    return BasePage.getPage(NavigationPage.class).navigateProfiles();
+    return getPage(NavigationPage.class).navigateProfiles();
```

Comment:
Method name should begin from the lower case.
```suggestion
    return getPage(navigationPage.class).navigateProfiles();
```